src/components/ItemEditorPage/LeftPanel/Items/Items.tsx

```diff
@@ -42,20 +42,20 @@ export default class Items extends React.PureComponent<Props, State> {
   }
 
   componentDidMount() {
-    const { items, selectedItemId, onSetItems, onSetReviewedItems } = this.props
-    const initialActiveItem = items.find(item => item.id === selectedItemId)
+    const { items, onSetReviewedItems } = this.props
     onSetReviewedItems(items)
-    if (initialActiveItem) {
```

Comment:
Moved this logic to the `componentDidUpdate` because when redirecting from the `creation flow,` the component `Items` is mounted before fetching the new item.